content/SCALE/SCALETutorials/Shares/NVMe-oF/_index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "NVMe-oF Subsystems"
+description: "Provides information on NVMe-oF subsystems and instruction on creating a subsystem and setting up enterprise configurations of subsystems."
+geekdocCollapseSection: true
+weight: 90
+related: false
+aliases:
+ -
+tags:
+- nvme-of
+- nvme
+- nvme baseNQN
+- block shares
+- subsystems
+- namespaces
+- hosts
+- ports
+- rdma
+- ana
+- asymmetric namespace access
+- remote direct memory access
+- cross-port referrals
+---
+
+## Overview of NVMe-oF
+
+{{< include file="/static/inclues/NVMe-oF-Overview.md" >}}
+
+## Configuring the NVMe-oF Service
+
+You can access the NVMe-of service screen from the:
+* <span class="material-icons">more_vert</span> dropdown menu on the **NVMe-oF Subsystems** widget on the **Shares** screen
+* **Global Configuration** button at the top of the **NVMe-oF** screen
+* **NVMe-oF** service option on the **System > Services** screen
+
+The **NVMe-oF Global Configuration** shows the base NQN for the service.
+
+{{< expand "What is the base NQN?" "v" >}}
+The *base NQN* us the standardized NVMe Qualified Name for the service.
+The NQN standard structure follows the format defined in the base specification. The maximum length/size is 223 bytes.
+TrueNAS subsystems use the base NQN as the root identifier for the NVMe subsystems in fabric deployments.
+Discovery controllers use the standardized NQN to advertise available subsystems.
+Storage systems use the base NQN to authenticate and authorize host connections.
+The base NQN format provides the foundation for all NVMe-oF naming, ensuring interoperability, and preventing naming conflicts across different vendors and implementations.
+{{< /expand >}}
+
+{{< trueimage src="/images/SCALE/Shares/NVMeoFGlobalConfigurationScreen.png" alt="NVMe-oF Global Configuration Screen" id="NVMe-oF Global Configuration Scree" >}}
+
+TrueNAS populates the **baseNQN** field with the NVMe identifier.
+Accept this value or click in the field to copy/paste a new, properly formatted base NQN identifier.
+NQN is used as the prefix when creating a subsystem, if a subnqn is not supplied.
+Modifying this value does not change the subnqn of any existing subsystems.
+
+Select **Generate Cross-port Referrals for Ports on This System**  to allow xport_referral.
```

Comment:
`xport_referral` is **our** implementation detail/nomenclature - it's not going to convey anything to a user.  Maybe instead something along the lines of

> Optionally select **Generate Cross-port Referrals for Ports on This System** if subsystems are published through multiple ports, and `connect-all` functionality is desired by clients.